.github/workflow-templates/browser-testing.yml

```
@@ -0,0 +1,63 @@
+name: Browser Testing
+
+on: [push]
+
+jobs:
+  run:
+    name: Browser
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: true
+      matrix:
+        browser:
+          - chrome
+          - firefox
+        os:
+          - ubuntu-latest
+          - windows-latest
+
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
```

Curious if dependabot will pick up on workflows in this directory 🤔 
```suggestion
        uses: actions/checkout@v3
```